src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -595,14 +595,6 @@ define(function (require, exports, module) {
                 </ul>
             );
 
-            var layerCount = (
-                <div
-                    title={strings.TOOLTIPS.LAYER_COUNT}
-                    className="layer-count">
-                    {doc.layers.selected.size}<span className="text-fancy"> oƒ </span>{doc.layers.count}
```

Comment:
We could probably rip out the associated `count` derived property, since I'm pretty sure it was only being used there.